### PR TITLE
Update mudlet-mapper.xml

### PR DESCRIPTION
**Pull request by Asteria-UI**
Additions to the mapping script for use in the Asteria mud.

* Login trigger (set mmp.game)
* Login script (registers envids, sets custom env colors for mapper.)
* Added Asteria to the update script for crowdmap.
* Added Asteria to 'true' check to display 'use crowdmap' mconfig.
* Added checks for balance in speedwalking. Had to game check asteria because of conflicts between existing gmcp from another game.
* Added 'open <dir> setting to mmp.openDoor() - game checks Asteria, else uses standard 'open door <dir>'
* Added trigger for door in the way.
* Game check on indoor/outdoor status tag for room creation. Asteria ignores this, works as expected for anyone else.

### CODE REVIEW

File: mudlet-mapper.xml [hunk 1]
```diff
@@ -4132,7 +4171,7 @@ mmp.settings:setOption(matches[3], val)</script>
 				<packageName></packageName>
 				<regex>^area list$</regex>
 			</Alias>
-			<AliasGroup isActive="no" isFolder="yes">
+			<AliasGroup isActive="yes" isFolder="yes">
```
Comment (keneanung) [inline mudlet-mapper.xml:4174]: We don't want to have mapping commands enabled on update:
```suggestion
			<AliasGroup isActive="no" isFolder="yes">
```

File: mudlet-mapper.xml [hunk 2]
```diff
@@ -6168,6 +6228,12 @@ end
 -- if we can't move, setup a polling timer to prompt walking when we can again.
 -- popular systems that expose balance &amp; equilibrium values can be added here as well, perhaps though a similarly-named function.
 
+if mmp.game == "asteria" then
```
Comment (keneanung) [inline mudlet-mapper.xml:6231]: I'm not sure this works... The script is ran on startup, at which point the game name is not yet set...

File: mudlet-mapper.xml [hunk 3]
```diff
@@ -10170,6 +10236,7 @@ function mmp_mapping_newroom(_, num)
               "."
           end
           -- check indoors status
+          if mmp.game ~= "asteria" then
```
Comment (keneanung) [inline mudlet-mapper.xml:10239]: There seems to be some indentation missing here...

**Review by keneanung — COMMENTED** [review]
Great that you got this working :thumbsup:. I have some comments:

File: mudlet-mapper.xml (hunk 1 above)
Comment (Asteria-UI) [reply mudlet-mapper.xml:4174]: Whoops! That was oversight on my part.

File: mudlet-mapper.xml (hunk 2 above)
Comment (Asteria-UI) [reply mudlet-mapper.xml:6231]: You're right, it didn't. I've added a function that'll hook onto the same login event that sets game envids to define this function for Asteria.

File: mudlet-mapper.xml (hunk 3 above)
Comment (Asteria-UI) [reply mudlet-mapper.xml:10239]: Noted! Perhaps even mended.

**Review by keneanung — APPROVED** [review]
Looks good for me now!

**Comment by Asteria-UI** [comment]
Now that the suggested changes have been approved, is there anything more I need to do before the PR can be merged?

**Pull request merged**